.dockerignore

```diff
@@ -11,7 +11,7 @@ solidity/node_modules/
 solidity/build/
 solidity/.python-version
 
-sgx/target/
+**/target/
```

Comment:
Don't think this change is needed anymore, all files should now go into `sgx/target`